frontend/public/components/edit-yaml.jsx

```diff
@@ -324,6 +324,12 @@ export const EditYAML_ = connect(stateToProps)(
           return;
         }
 
+        // If this is NOT a namespaced resource and namespace is provided, display an error.
+        if (obj.metadata.namespace && !model.namespaced) {
+          this.handleError('"metadata.namespace" field found in YAML is not necessary.');
```

Comment:
Wondering if we remove the `namespace` key instead?  The CLI appears to do that.

Reply:
> Wondering if we remove the `namespace` key instead? The CLI appears to do that.

Yeah, I had the same thought. I think that's a reasonable fix since it's consistent with the CLI. The API server also ignores unknown fields.